docs(tutorials): ceph cluster-MTA-1484 (#117)
scaleway/docs-content: tutorials/ceph-cluster/index.mdx

  sudo apt update
  ```
5. Type the following command to install `ceph-deploy`:

  ```
  sudo apt install ceph-deploy
  ```

## Creating a ceph-deploy user

`ceph-deploy` needs access to a user that has passwordless sudo privileges to be able to install software and configuration files on the storage nodes.
Ceph deploy will be able to connect as any user, including the `root` user, however this is not recommended. To simplify the software configuration, it is recommended to use the same username on all Ceph nodes in the cluster.

<Message type="important">

 Do not use `ceph` as login for your user, as it is reserved for Ceph-daemons. The installation of Ceph will fail if a `ceph` user exists on a machine.

</Message>

1. Type the following command to connect to your Ceph node using SSH:

  ```
  ssh user@ceph-node
  ```

2. Type the following command to create a user called `ceph-deploy`: 

  ```
  sudo useradd -d /home/ceph-deploy -m ceph-deploy
  ```
 
  <Message type="note">

   You can rename the user to your own preferences if required.

  </Message>

3. Type the following command to configure the password of the `ceph-deploy` user:

  ```
  sudo passwd ceph-deploy
  ```

> 4. Type the following command to add the user to the sudoers configuration so it has sudo rights:

Reviewer: ```suggestion
4. Type the following command to add the user to the sudoers configuration so they have sudo rights:
```

  ```
  echo "ceph-deploy ALL = (root) NOPASSWD:ALL" | sudo tee /etc/sudoers.d/ceph-deploy
  sudo chmod 0440 /etc/sudoers.d/ceph-deploy
  ```

5. Type the following command to install an NTP client on your Ceph nodes:

  ```
  sudo apt install ntpsec
  ```

  <Message type="note">

>    The installation of an NTP client allows you to avoid synchronization issues caused by time-drift.

Reviewer 2: Maybe add a wikipedia link to ntp ? https://en.wikipedia.org/wiki/Network_Time_Protocol

>    The installation of an NTP client allows you to avoid synchronization issues caused by time-drift.

Author: ```suggestion
   The installation of an [NTP](https://en.wikipedia.org/wiki/Network_Time_Protocol) client allows you to avoid synchronization issues caused by time-drift.
```

   </Message>

6. Install Python to deploy the cluster:

  ```
  sudo apt install python-minimal
  ```

> 7. Repeat all the previous steps 3 times to setup each of your 3 nodes.

Reviewer: ```suggestion
7. Repeat all the previous steps 3 times to set up each of your 3 nodes.
```

  mkdir my-ceph-cluster
  cd my-ceph-cluster
  ```

2. Type the following command to create the cluster:

    ```
    ceph-deploy --username ceph-deploy new ceph-node-a
    ```

    <Message type="note">

      Replace `ceph-node-a` with the FQDN of your node, for example: `ceph-node-a.example.com`.

    </Message>

    
    <Message type="tip">

     * If you have more than one network interface on the machine, type the following command to add the public network setting under the `[global]` section of your Ceph configuration file: 
 
      ```
      public network = <ip-address>/<netmask>
      ```
      For example:

      ```
      public network = 62.210.16.23/24
      ```

     * Refer to the [official documentation](https://docs.ceph.com/docs/master/rados/configuration/network-config-ref/) for more information on the network configuration.

    </Message>

3. Type the following command to install the ceph packages on the nodes:

  ```
  ceph-deploy --username ceph-deploy install ceph-node-a ceph-node-b ceph-node-c
  ```

4. Type the following command to deploy the initial monitor(s) and gather the keys:

  ```
  ceph-deploy --username ceph-deploy mon create-initial
  ```

>  Once completed, you should have the following files in your directory, verify that they are generated using the `ls` command:

Reviewer: ```suggestion
 Once completed, you should have the following files in your directory. Verify that they are generated using the `ls` command:
```

    ```
    ceph.bootstrap-mds.keyring  ceph.bootstrap-rgw.keyring  ceph-deploy-ceph.log
    ceph.bootstrap-mgr.keyring  ceph.client.admin.keyring   ceph.mon.keyring
    ceph.bootstrap-osd.keyring  ceph.conf
    ```

5. Copy the configuration file and admin key to your Ceph Nodes using `ceph-deploy`:

  ```
  ceph-deploy --username ceph-deploy admin ceph-node-a ceph-node-b ceph-node-c
  ```

  <Message type="note">

   This step allows you to use the ceph CLI without having to specify the monitor address and `ceph.client.admin.keyring` each time you execute a command.

  </Message>

6. Type the following command to deploy the manager daemon on all Ceph nodes:

  ```
  ceph-deploy --username ceph-deploy mgr create ceph-node-a ceph-node-b ceph-node-c
  ```

  <Message type="note">

>    These daemons operate in an active/standby pattern. Deploying multiple manager daemons ensures in case one of the daemons fail for whatever reason, another one can take over without interrupting service.

Reviewer: ```suggestion
   These daemons operate in an active/standby pattern. Deploying multiple manager daemons ensures that if one of the daemons fails for any reason, another one can take over without interrupting service.
```
